Runtime/Core/BuiltinMaterials.cs

```diff
@@ -299,7 +301,9 @@ internal static Material ShapePreviewMaterial
         {
             get
             {
-                Init();
+                if(s_ShapePreviewMaterial == null)
```

Comment:
This doesn't seem to be used anymore.

Reply:
Good catch!
I changed that in [b8986d0](https://github.com/Unity-Technologies/com.unity.probuilder/pull/500/commits/b8986d02ec80631902357b6b234a9691a16b9125) so that the draw shape tool actually uses that material.
Also removed another unused material in [1ae1f31](https://github.com/Unity-Technologies/com.unity.probuilder/pull/500/commits/1ae1f318c0c36e275564f74d893684ca24308b40)